src/Api/Configuration/BtmsOptions.cs

```diff
@@ -9,10 +9,12 @@ public class BtmsOptions
     public required string BaseUrl { get; init; }
 
     [Required]
-    public required string Username { get; init; }
+    public required string Password { get; init; }
+
+    public bool StubEnabled { get; init; } = false;
 
     [Required]
-    public required string Password { get; init; }
+    public required string Username { get; init; }
```

Comment:
Nit - is this a full reformat too as the order has changed? Also need to watch this as it's just noisy

Reply:
No, this was me.